src/styles/base/_03-mixins.scss

```diff
@@ -58,8 +49,6 @@
 @mixin breakout($v-padding: var(--space)) {
     margin-left: calc(-50vw + 50%);
```

Comment:
The breakout mixin in D8 also adds 0.5rem of padding to these margins:
```
margin-left: calc(0.5rem + calc(-50vw + 50%));
margin-right: calc(0.5rem + calc(-50vw + 50%));
```
I think it is because the viewport calculation doesn't take into account the scrollbar (if one is present), triggering an x overflow?

Reply:
You mean a browser scrollbar? Is this specifically for older browsers? Sounds so odd, because I'm not seeing that on my machine...

Reply:
It's an odd one, but there seems to be a difference when a browser presents an overlay scrollbar vs the standard scrollbar.

Overlay Scrollbar

<img width="1377" alt="overlay-scrollbar" src="https://user-images.githubusercontent.com/1325185/92641397-b00a8e80-f2ac-11ea-9f34-014ccddba9e7.png">

Non-overlay Scrollbar

![non-overlay-scrollbar](https://user-images.githubusercontent.com/1325185/92641426-ba2c8d00-f2ac-11ea-8666-8f3ab2512055.png)

The screenshots are from Chrome, but I was able to reproduce the same in Firefox also.

What is interesting, is that I only see a non-overlay scrollbar when viewing a browser window in my external monitor. This leads to me believe that it might be a setting related to pixel density?

Either way, the non-scrollbar overlay gutter does not get calculated in the VW unit. [This article](https://cloudfour.com/thinks/breaking-out-with-viewport-units-and-calc/#one-big-dumb-caveat) seems to confirm that.

I think that might be why the breakout mixin in D8 adds the extra 0.5rem to the calc.

Anyway, I'd be fine with removing that extra 0.5rem as you did here, as I don't think we can control the non-overlay vs overlay scrollbar, as far as I can tell.

How do you want to account for the differences in the wrapper/breakpoint mixins in the Design System vs D8? Should we make the changes there, as well?

Reply:
Ahhh, thanks for writing this up, it helped make it really clear.

I think instead _I'm_ going to add back in the margins here—otherwise it sounds like I'd be introducing a regression. I will also add a comment, because wowza, what an odd one